app/controllers/reservations_controller.rb

```diff
@@ -330,7 +330,7 @@ def checkout # rubocop:disable all
     # prep for receipt page and exit
     @check_in_set = []
     @check_out_set = checked_out_reservations
-    render('receipt') && return
+    render('receipt', layout: 'application_with_search_sidebar') && return
```

Comment:
Ok, so this makes the reservation show view a little cramped... any thoughts?

![selection_076](https://cloud.githubusercontent.com/assets/901808/12737235/c89f4ca8-c924-11e5-9e43-f2d4af269f06.png)